include/xtensor/xtensor_simd.hpp

```diff
@@ -62,15 +63,64 @@ namespace xt_simd
     template <class T>
     using revert_simd_type = xsimd::revert_simd_type<T>;
 
-    using xsimd::set_simd;
-    using xsimd::load_simd;
-    using xsimd::store_simd;
-    using xsimd::select;
-    using xsimd::get_alignment_offset;
+
+    template <class T, class requested_type=T>
+    inline simd_type<requested_type> load_simd(const T* src, aligned_mode)
+    {
+        using arch_type = xsimd::default_arch;
+        return xsimd::batch<requested_type, arch_type>::load_aligned(src);
```

Comment:
Here it should be `xsimd::batch<xsimd::simd_return_type<T, requested_type>>::load_aligned.`

That would fix the code. However, I would like to find a better way in xsimd to hide that behind a nice API, just as before.